Makefile

```diff
@@ -77,6 +77,13 @@ test-interceptor: check-go121-install check-jq-install check-vault-install build
 	@echo "Running e2e tests for interceptor..."
 	cd interceptor && ./test/e2e.sh
 
+.PHONY: run-local-interceptor
+run-local-interceptor: build-interceptor ## Create placeholder variables for interceptor local environment and run 
+	@echo
+	@echo "Warning: The run-local-interceptor's variables are placeholders and can be set to stage env variables with either test/set_stage_env.sh or using the E2E script."
+	@echo
+	CAD_SILENT_POLICY=test CAD_PD_TOKEN=test TOKEN=test ./bin/interceptor
```

Comment:
Should `TOKEN` be called `PD_SIGNATURE`? Looking at https://github.com/openshift/configuration-anomaly-detection/pull/363/files 

We'll probably want to wait for https://github.com/openshift/configuration-anomaly-detection/pull/363/files before this merges anyway, correct?